test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -480,6 +480,8 @@ func testWebhookAdmission(t *testing.T, watchCache bool) {
 		"--disable-admission-plugins=ServiceAccount,StorageObjectInUseProtection",
 		// force enable all resources so we can check storage.
 		"--runtime-config=api/all=true",
+		// enable feature-gates that protect resources to check their storage, too.
+		"--feature-gates=EphemeralContainers=true",
```

Comment:
just to verify, adding this made the test fail until the conversion was done, right?

Reply:
That's correct, enabling the feature gate made the test fail without the conversion, which I thought was impressive.